docs/content/node-operation/past-sporks.mdx

```diff
@@ -381,6 +381,18 @@ While FCL v0.0.67 is not strictly necessary to use Mainnet 6, we strongly recomm
 
 ## Testnet Sporks
 
+## Devnet38
+
+| Spork Info                                                                              |
+|:----------------------------------------------------------------------------------------|
+| **Date**: Oct 19th, 2022                                                                |
+| **Root Height**: 83007730                                                               |
+| **Root Parent ID**: 1b914363c9a34c46b93974adeefb10c546578d7f3f4ac9291e01d746d2c84226    |
+| **Root State Commit**: 79df428bc27f22d38c233777610d93d33e180f23cfbc640a95d144a508e0f080 |
+| **Git Commit**: 2373799f83f2744100dc29ca11bce2a13c3cb5e9                                |
+| **Branch/Tag**: v0.28.3                                                                 |
```

Comment:
Let's use v0.28.4

Reply:
ok lemme build the `without_netgo` images for it.

Reply:
updated